Code/Core/Strings/LevenshteinDistance.cpp

```diff
@@ -37,89 +37,140 @@ namespace
 }
 
 // Levenshtein distance
-// https://en.wikibooks.org/wiki/Algorithm_Implementation/Strings/Levenshtein_distance#C
+// https://github.com/gitster/git/blob/master/levenshtein.c
```

Comment:
I took a look at the licensing terms of this source code and it seems to be GPLv2. Unfortunately, that makes it incompatible with the more permissive licensing of FASTBuild's (which for example allows for commercial use without source code release).